Modules/OptionHolder.cs

```diff
@@ -2038,6 +1907,27 @@ public static void SetupAdtRoleOptions(int id, CustomRoles role, CustomGameMode
             .SetHidden(!canSetChance)
             .SetGameMode(customGameMode) as IntegerOptionItem;
 
+        if (teamSpawnOptions)
+        {
+            var impOption = BooleanOptionItem.Create(id + 3, "ImpCanBeRole", true, tab, false)
+                .SetParent(spawnOption)
+                .SetGameMode(customGameMode)
+                .AddReplacement(("{role}", role.ToColoredString()));
+
+            var neutralOption = BooleanOptionItem.Create(id + 4, "NeutralCanBeRole", true, tab, false)
+                .SetParent(spawnOption)
+                .SetGameMode(customGameMode)
+                .AddReplacement(("{role}", role.ToColoredString()));
+
+            var crewOption = BooleanOptionItem.Create(id + 5, "CrewCanBeRole", true, tab, false)
+                .SetParent(spawnOption)
+                .SetGameMode(customGameMode)
+                .AddReplacement(("{role}", role.ToColoredString()));
```

Comment:
there are suspicions that some add-ons will have duplicate id's in the logs

Reply:
That is not the case. Because all of them starts at +10 or above. And this used the ids below that, I have ofc tested loading it in, and seeing no conflictions.